sass/_universal.scss

```diff
@@ -0,0 +1,40 @@
+@import '_variables.scss';
+@import '_mixins.scss';
+/* UNIVERSAL */
+
+html {
+	box-sizing: border-box;
+
+	height: 100%;
+}
+
+*,
+*:before,
+*:after {
+	box-sizing: inherit;
+}
+
+html,
+body {
+	@include fontfamily(sans);
+}
+
+body {
+	min-width: $sm-mq;
+}
+
+// for bits like the close button on a modal, seems like this should be elsewhere...
+.close {
+	cursor: pointer;
+}
+
+.visible {
+	display: inline-block;
+}
+
+// get rid of that nasty space under an image when it's wrapped in an anchor
+.img {
+	a {
```

Comment:
No, the ampersand after the selector is a-okay, it bumps that selector to be the parent.

**1)** ampersand + space + selector:
```css
.foo {
  & .bar {}
}
```
**2)** lone selector
```css
.foo {
  .bar {}
}
```
**3)**  selector + ampersand:
```css
.foo {
  .bar& {}
}
```
**4)** selector + space + ampersand:
```css
.foo {
  .bar & {}
}
```

---

Numbers **1** & **2** result in this:
```css
.foo .bar {}
```
Number **3** throws an error.
Number **4** results in this:
```css
.bar .foo {}
```